tools/slang-test/render-api-util.h

```diff
@@ -0,0 +1,61 @@
+#ifndef SLANG_RENDER_API_UTIL_H
+#define SLANG_RENDER_API_UTIL_H
+
+#include "../../source/core/slang-string.h"
+#include "../../source/core/slang-result.h"
+
+
+enum class RenderApiType
+{
+    kUnknown = -1,
+    kOpenGl = 0,
+    kVulkan,
+    kD3D12,
+    kD3D11,
+    kCountOf,
+};
+
+// Use a struct wrapped Enum instead of enum class, cos we want to be able to manipulate as integrals
```

Comment:
This is maybe a case of me not knowing English as well as I think. I'm aware that we can say "an integral number" to mean an integer, but I don't recall seeing just "integral" used as a synonym for "integer."

Reply:
Think that's okay - 'Expressed or expressible as or in terms of integers.'
https://www.thefreedictionary.com/integral